src/volatile-root/volatile-root.c

```diff
@@ -18,13 +18,13 @@
 #include "string-util.h"
 #include "volatile-util.h"
 
-static int make_volatile(const char *path) {
```

Comment:
For the first and the third commit: "rename with" is not grammatically correct. "rename to" is the right verb.